main.tf

```
@@ -51,14 +51,14 @@ resource "helm_release" "fluent_bit" {
   chart      = "fluent-bit"
   repository = "https://fluent.github.io/helm-charts"
   namespace  = kubernetes_namespace.logging.id
-  version    = "0.16.4"
+  version    = "0.36.0"
   timeout    = 1500
 
   values = [templatefile("${path.module}/templates/fluent-bit.yaml.tpl", {
     elasticsearch_host              = var.elasticsearch_host
     elasticsearch_modsec_audit_host = var.elasticsearch_modsec_audit_host
     cluster                         = terraform.workspace
-    fluentbit_app_version           = "1.8.4" # Pinned to version, because of this issue https://github.com/fluent/fluent-bit/issues/4260
+    fluentbit_app_version           = "2.1.7" # Pinned to version, because of this issue https://github.com/fluent/fluent-bit/issues/4260
```

```suggestion
    fluentbit_app_version           = "2.1.7"
```

The references issue is now closed, so we needn't mention the pin anymore 👍🏼